core/domain/story_domain_test.py

```diff
@@ -625,6 +645,18 @@ def test_add_node_validation(self) -> None:
             'next node id for the story'):
             self.story.add_node('node_4', 'Title 4')
 
+    def test_delete_node_validation(self) -> None:
+        self.assertEqual(len(self.story.story_contents.nodes), 2)
+        with self.assertRaisesRegex(
+            ValueError,
+            'The node with id %s is the starting node for the story, '
+            'change the starting node before deleting it.' % self.NODE_ID_1):
+            self.story.delete_node(self.NODE_ID_1)
```

Comment:
```suggestion
            'change the starting node before deleting it.' % self.NODE_ID_1
        ):
            self.story.delete_node(self.NODE_ID_1)
```

Please indent so that it's clear where the `with` argument ends and the following block begins

Reply:
done